src/assets/stack-blitz/package.json

```diff
@@ -17,9 +17,11 @@
     "@angular/core": "~14.0.0",
     "@angular/forms": "~14.0.0",
     "@angular/material": "${version}",
+    "@angular/material-moment-adapter": "${version}",
     "@angular/platform-browser": "~14.0.0",
     "@angular/platform-browser-dynamic": "~14.0.0",
     "@angular/router": "~14.0.0",
+    "moment": "2.18.1",
```

Comment:
Should we set this to `^2.18.1` to be a bit more flexible?

Reply:
ohh yeah, that's right, I changed it to `^2.18.1`.